src/components/experiment-tracking/run-metadata/run-metadata.js

```diff
@@ -98,109 +101,132 @@ const RunMetadata = ({
                 </tr>
-                <tr>
-                  {i === 0 ? <td>Creation Date</td> : null}
-                  <td>{`${humanReadableTime} (${sanitiseEmptyValue(
+              </tbody>
+            ) : null}
+          </>
+        ))}
+        <TransitionGroup
+          component={'tbody'}
+          className="details-metadata__run--wrapper"
+        >
+          {runs.map((run, i) => {
+            const humanReadableTime = toHumanReadableTime(run.id);
+
+            return (
+              <CSSTransition
+                key={run.id}
+                timeout={300}
+                classNames={'details-metadata__run-animation'}
+                enter={isSingleRun ? false : true}
+                exit={isSingleRun ? false : true}
+              >
+                <tr
+                  className={classnames('details-metadata__run', {
+                    'details-metadata__run--first-run': i === 0,
+                    'details-metadata__run--first-run-comparision-view':
+                      i === 0 && enableComparisonView,
+                  })}
+                >
+                  <td className="details-metadata__title">
+                    <span
+                      className="details-metadata__title-detail"
+                      onClick={() => onTitleOrNoteClick(run.id)}
+                      title={sanitiseEmptyValue(run.title)}
+                    >
+                      {sanitiseEmptyValue(run.title)}
+                    </span>
+
+                    <ul className="details-metadata__buttons">
+                      <IconButton
+                        active={run.id === pinnedRun}
+                        ariaLive="polite"
+                        className={classnames(
+                          'pipeline-menu-button--labels',
+                          'pipeline-menu-button__pin',
+                          {
+                            'details-metadata__buttons--selected-pin':
+                              run.id === pinnedRun,
+                          }
+                        )}
+                        icon={
+                          run.id === pinnedRun ? SelectedPin : UnSelectedPin
+                        }
+                        labelText={
+                          run.id === pinnedRun ? 'Baseline' : 'Make baseline'
+                        }
+                        labelTextPosition="bottom"
+                        onClick={() => setPinnedRun(run.id)}
+                        visible={enableShowChanges}
+                      />
+                      <HiddenMenu isBookmarked={run.bookmark} runId={run.id} />
+                      <IconButton
+                        ariaLive="polite"
+                        className="pipeline-menu-button--labels__close"
+                        icon={CloseIcon}
+                        labelText="Remove run"
+                        labelTextPosition="bottom"
+                        onClick={() => onRunSelection(run.id)}
+                      />
```

Comment:
These shouldn't be here when comparison mode is active but only one run is selected.